app/views/components/_govuk_notification_banner.html.erb

```diff
@@ -0,0 +1,67 @@
+<%
+  if local_assigns[:type] ==  "success"
+    success_banner = true
+    type_class = "govuk-notification-banner--" + local_assigns[:type]
+  end
```

Comment:
You might want to call `.to_s` on the type as the current banner component does: https://github.com/UKGovernmentBEIS/govuk-design-system-rails/blob/master/app/views/components/_hmcts_banner.html.erb#L2, which allows it to be used with `flash` types which are a `symbol`. Although this would be a deviation from the spec so up to you!